lib/jobs/dell-wsman-simple-update-firmware.js

```diff
@@ -0,0 +1,249 @@
+// Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+
+'use strict';
+
+var di = require('di');
+var DOMParser = require('xmldom').DOMParser;
+
+module.exports = WsmanSimpleFirmwareUpdateJobFactory;
+di.annotate(WsmanSimpleFirmwareUpdateJobFactory, new di.Provide('Job.Dell.Wsman.Simple.Update.Firmware'));
+di.annotate(WsmanSimpleFirmwareUpdateJobFactory, new di.Inject(
+    'Job.Dell.Wsman.Base',
+    'JobUtils.WsmanTool',
+    'Logger',
+    'Util',
+    'Assert',
+    'Errors',
+    'Promise',
+    'uuid',
+    'Services.Configuration',
+    'fs',
+    'ChildProcess'
+));
+
+function WsmanSimpleFirmwareUpdateJobFactory(
+    BaseJob,
+    WsmanTool,
+    Logger,
+    util,
+    assert,
+    errors,
+    Promise,
+    uuid,
+    configuration,
+    fs,
+    ChildProcess
+) {
+    var logger = Logger.initialize(WsmanSimpleFirmwareUpdateJobFactory);
+
+    /*
+     * @param {object} options
+     * @param {object} context
+     * @param {string} taskId
+     * @constructor
+     */
+    function WsmanSimpleFirmwareUpdateJob(options, context, taskId) {
+        var self = this;
+        WsmanSimpleFirmwareUpdateJob.super_.call(self, logger, options, context, taskId);
+        assert.object(self.options);
+        logger.debug('Taskgraph options: ' + JSON.stringify(self.options));
```

Comment:
Actually there is no need to print this message, the task scheduler will print the options defaultly.

Reply:
Got it